app/src/util/tools.ts

```diff
@@ -354,6 +354,52 @@ export const formatNumber = (number: string, decimals = 2): string => {
   return `${formattedSubstring}${decimals > 0 ? '.' + fixedInt.split('.')[1] : ''}`
 }
 
+export const formatHistoryDate = (dateData: string): string => {
+  const date = new Date(dateData)
+  const minute = date.getMinutes()
+  const minuteWithZero = (minute < 10 ? '0' : '') + minute
+  const hour = date.getHours()
+  const hourWithZero = (hour < 10 ? '0' : '') + hour
+  return `${date.getDate()}.${date.getMonth()} - ${hourWithZero}:${minuteWithZero}`
```

Comment:
Also shouldn't the date string be formatted as dd/mm instead of dd.mm ? Its not obvious to me that value represents a date.

Reply:
You mean with the dash? Well i was basing it on figma designs here is how it is there...
![Screenshot 2021-01-13 at 15 23 41](https://user-images.githubusercontent.com/33226956/104464587-67911380-55b3-11eb-8f68-00244a357f01.png) 
- Here is the figma link it should be on right side https://www.figma.com/file/R056tSYMjvfwaFidEBItYy/Widgets-V2?node-id=1209%3A141